### DISCUSSION TITLE
¿Cómo saltarse la pantalla inicial para cambiar el nombre al hacer login?

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#150] [2. Questions and Answers] [answered]
Buenas a todos,

Quisiera saber cómo se puede saltar la pantalla de cambio de nombre al hacer login en una compañía. En algunas implementaciones no hace sentido que **siempre** se acceda a esa vista antes de poder interactuar con la vista de grupos y canales.

Saludos.

**Comment by acedward** [comment, marked as answer]
1. En este momento no es posible, ni está en el roadmap esta funcionalidad.
2. La razón de esto, es que esperamos que los usuarios no hagan logout, ni se cambien de dispositivo con frecuencia.

**Reply by acedward** [reply]
@SeIturra algún feedback adicional? sino para cerrar el ticket
